commit 68a889b8b7f9555a8b69c7df1f99d3e498ccf3e6 (refs/changes/71/3271/2)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-01-26T12:15:57-05:00 (1 year, 8 months ago)
    
    TRITON-88 node-triton "env" doesn't call its callback
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index 03ec7dc..cecd8a0 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -14,6 +14,7 @@ Known issues:
   `triton network ip update`
 - [TRITON-24] node-triton ListNetworkIPs has unordered results, e.g.
   `triton network ip list NETWORK`
+- [TRITON-88] node-triton "env" doesn't call its callback
 
 ## 5.5.0
 
diff --git a/lib/do_env.js b/lib/do_env.js
index f6ba08e..69d579a 100644
--- a/lib/do_env.js
+++ b/lib/do_env.js
@@ -164,6 +164,7 @@ function do_env(subcmd, opts, args, cb) {
         (shortOpts ? ' -'+shortOpts : ''),
         (profile.name === this.tritonapi.profile.name
             ? '' : ' ' + profile.name));
+    cb();
 }
 
 do_env.options = [
